algorithms/linfa-svm/src/regression.rs

```diff
@@ -153,10 +153,10 @@ macro_rules! impl_regression {
     };
 }
 
-impl_regression!(Array2<f32>, Array2<f32>, f32);
-impl_regression!(Array2<f64>, Array2<f64>, f64);
-impl_regression!(ArrayView2<'_, f32>, ArrayView2<'_, f32>, f32);
-impl_regression!(ArrayView2<'_, f64>, ArrayView2<'_, f64>, f64);
+// impl_regression!(Array2<f32>, Array2<f32>, f32);
+// impl_regression!(Array2<f64>, Array2<f64>, f64);
```

Comment:
I commented out these tests, since the macro calls `try_single_target`. Should we remove them?

Reply:
I'm pretty sure those are impls used by the rest of the library, not tests. This fitting algorithm is clearly single-target, so just remove the impls with 2D targets and leave the other ones.